lib/savings_screen/view/savings_page.dart

```diff
@@ -0,0 +1,362 @@
+                            style: Theme.of(context)
+                                .textTheme
+                                .displaySmall
+                                ?.copyWith(
+                                  color: const Color(0xffABABAB),
+                                  fontWeight: FontWeight.w400,
+                                  fontFamily: 'open sans',
+                                  fontSize: 12,
+                                ),
+                          ),
+                        ],
+                      ),
+                      const SizedBox(
+                        height: 10,
+                      ),
+                      const Divider(
+                        height: 0.5,
+                        color: Color(0xffE7E7E7),
+                        thickness: 0.5,
+                      ),
+                      const SizedBox(
+                        height: 10,
+                      ),
+                      Column(
+                        // ignore: avoid_redundant_argument_values
+                        mainAxisAlignment: MainAxisAlignment.start,
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Row(
+                            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                            children: [
+                              Text(
+                                'New Drone',
+                                style: Theme.of(context)
+                                    .textTheme
+                                    .displaySmall
+                                    ?.copyWith(
+                                      color: AppColors.secondaryColorW,
+                                      fontWeight: FontWeight.w400,
+                                      fontFamily: 'open sans',
+                                      fontSize: 14,
+                                    ),
+                              ),
+                              Text(
+                                r'$ 79.45',
+                                style: Theme.of(context)
+                                    .textTheme
+                                    .displaySmall
+                                    ?.copyWith(
+                                      color: AppColors.secondaryColorW,
+                                      fontWeight: FontWeight.w600,
+                                      fontFamily: 'open sans',
+                                      fontSize: 14,
+                                    ),
+                              ),
+                            ],
+                          ),
+                          const SizedBox(
+                            height: 10,
+                          ),
+                          Text(
+                            'Blocked until November 1st 2022',
+                            style: Theme.of(context)
```

Comment:
This theme also seems to be repetitive, look at the next comment for details of what to do